Installed CMake and pkgconf files still link to libexiv2-xmp

These should have been removed after https://github.com/Exiv2/exiv2/pull/2187

@piponazo Don't know how to remove it from the exported CMake config though. If I try, I get
```
CMake Error: install(EXPORT "exiv2Config" ...) includes target "exiv2lib" which requires target "exi
v2-xmp" that is not in any export set.
```

--- a/releasenotes/Cygwin/ReadMe.txt
+++ b/releasenotes/Cygwin/ReadMe.txt
@@ -13,8 +13,8 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2.exe
-exiv2 dll                                       bin/cygexiv2-@VN@.dll
-link libraries                                  lib/libexiv2.dll.a & libexiv2-xmp.a
+shared library                                  bin/cygexiv2-@VN@.dll
+import library                                  lib/libexiv2.dll.a
 CMake support/consume files                     lib/cmake/exiv2
 pkg-config file                                 lib/pkgconfig/exiv2.pc
 man pages                                       share/man/

--- a/releasenotes/Darwin/ReadMe.txt
+++ b/releasenotes/Darwin/ReadMe.txt
@@ -13,9 +13,9 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2
-libraries                                       lib/libexiv2.@VM@.@VN@.@VD@@VR@.dylib & libexiv2-xmp.a
-pkg-config file                                 lib/pkgconfig/exiv2.pc
+shared library                                  lib/libexiv2.@VM@.@VN@.@VD@@VR@.dylib
 CMake support/consume files                     lib/cmake/exiv2
+pkg-config file                                 lib/pkgconfig/exiv2.pc
 include files                                   include/exiv2/
 man pages                                       share/man/
 localisation files                              share/locale/

--- a/releasenotes/Linux/ReadMe.txt
+++ b/releasenotes/Linux/ReadMe.txt
@@ -13,8 +13,7 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2
-libraries                                       lib/libexiv2.so.@VM@.@VN@.@VD@@VR@ & libexiv2-xmp.a
-pkg-config file                                 lib/pkgconfig/exiv2.pc
+shared library                                  lib/libexiv2.so.@VM@.@VN@.@VD@@VR@
 CMake support/consume files                     lib/cmake/exiv2
 pkg-config file                                 lib/pkgconfig/exiv2.pc
 man pages                                       share/man/

--- a/releasenotes/MSVC/ReadMe.txt
+++ b/releasenotes/MSVC/ReadMe.txt
@@ -13,8 +13,8 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2.exe
-exiv2 dll                                       bin/exiv2.dll
-link libraries                                  lib/exiv2.lib & exiv2-xmp.lib
+shared library                                  bin/exiv2.dll
+import library                                  lib/exiv2.lib
 CMake support/consume files                     lib/cmake/exiv2
 include files                                   include/exiv2/
 sample code                                     samples/exifprint.cpp

--- a/releasenotes/MinGW/ReadMe.txt
+++ b/releasenotes/MinGW/ReadMe.txt
@@ -13,7 +13,7 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2.exe
-shared library                                  bin/exiv2-@VN@.dll
+shared library                                  bin/libexiv2.dll
 import library                                  lib/libexiv2.dll.a
 CMake support/consume files                     lib/cmake/exiv2
 pkg-config file                                 lib/pkgconfig/exiv2.pc

--- a/releasenotes/MinGW/ReadMe.txt
+++ b/releasenotes/MinGW/ReadMe.txt
@@ -12,10 +12,9 @@ COPYING                                         GPLv2.0 Software License
 
 Deliverable                                     Location
 -----------                                     --------
-
 exiv2 and sample applications                   bin/exiv2.exe
-exiv2 dll                                       bin/msys-exiv2-@VN@.dll
-link libraries                                  lib/libexiv2.dll.a & libexiv2-xmp.a
+shared library                                  bin/exiv2-@VN@.dll
+import library                                  lib/libexiv2.dll.a
 CMake support/consume files                     lib/cmake/exiv2
 pkg-config file                                 lib/pkgconfig/exiv2.pc
 man pages                                       share/man/

--- a/releasenotes/Unix/ReadMe.txt
+++ b/releasenotes/Unix/ReadMe.txt
@@ -13,7 +13,7 @@ COPYING                                         GPLv2.0 Software License
 Deliverable                                     Location
 -----------                                     --------
 exiv2 and sample applications                   bin/exiv2
-libraries                                       lib/libexiv2.so.@VM@.@VN@.@VD@ & libexiv2-xmp.a
+shared library                                  lib/libexiv2.so.@VM@.@VN@.@VD@
 CMake support/consume files                     lib/cmake/exiv2
 pkg-config file                                 lib/pkgconfig/exiv2.pc
 man pages                                       share/man/

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -189,7 +189,6 @@ endif()
 
 if (EXIV2_ENABLE_XMP)
     target_link_libraries(exiv2lib PRIVATE exiv2-xmp)
-    list(APPEND libs_private_list "exiv2-xmp")
 elseif(EXIV2_ENABLE_EXTERNAL_XMP)
     target_link_libraries(exiv2lib PUBLIC ${XMPSDK_LIBRARY})
     target_include_directories(exiv2lib PUBLIC ${XMPSDK_INCLUDE_DIR})

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -188,7 +188,7 @@ if (EXIV2_ENABLE_XMP OR EXIV2_ENABLE_EXTERNAL_XMP)
 endif()
 
 if (EXIV2_ENABLE_XMP)
-    target_link_libraries(exiv2lib PRIVATE exiv2-xmp)
+    target_sources(exiv2lib PRIVATE $<TARGET_OBJECTS:exiv2-xmp>)
 elseif(EXIV2_ENABLE_EXTERNAL_XMP)
     target_link_libraries(exiv2lib PUBLIC ${XMPSDK_LIBRARY})
     target_include_directories(exiv2lib PUBLIC ${XMPSDK_INCLUDE_DIR})

--- a/xmpsdk/CMakeLists.txt
+++ b/xmpsdk/CMakeLists.txt
@@ -26,14 +26,10 @@ add_library(exiv2-xmp OBJECT
     include/XMP_Version.h
 )
 
-target_link_libraries(exiv2-xmp
-    PRIVATE 
-    EXPAT::EXPAT
-)
-
 target_include_directories(exiv2-xmp SYSTEM
-    PRIVATE 
+    PRIVATE
         ${PROJECT_SOURCE_DIR}/xmpsdk/include
+        ${EXPAT_INCLUDE_DIRS}
 )
 
 # Prevent a denial-service-attack related to XML entity expansion
@@ -41,7 +37,7 @@ target_include_directories(exiv2-xmp SYSTEM
 # See https://bugzilla.redhat.com/show_bug.cgi?id=888769
 target_compile_definitions(exiv2-xmp PRIVATE BanAllEntityUsage=1)
 
-if (MSVC)
+if (WIN32)
     target_compile_definitions(exiv2-xmp PRIVATE XML_STATIC)
 endif()
 
@@ -53,10 +49,3 @@ endif()
 if (BUILD_SHARED_LIBS)
     set_property(TARGET exiv2-xmp PROPERTY POSITION_INDEPENDENT_CODE ON)
 endif()
-
-# The private dependencies of a static library still need to be exported because they are needed to properly link the consumers of the static library.
-install(TARGETS exiv2-xmp EXPORT exiv2Config	
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}	
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}	
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-)

